pkg/sources/gitlab/gitlab.go

```diff
@@ -344,7 +355,11 @@ func (s *Source) getAllProjects(ctx context.Context, apiClient *gitlab.Client) (
 	for {
 		userProjects, res, err := apiClient.Projects.ListUserProjects(user.ID, projectQueryOptions)
 		if err != nil {
-			return nil, fmt.Errorf("received error on listing user projects: %w", err)
+			err = fmt.Errorf("received error on listing user projects: %w", err)
+			if err := reportErr(ctx, err); err != nil {
+				return nil, err
+			}
+			break
```

Comment:
Does using `break` instead of `return` mean that this error is now non-fatal when running with units?

Reply:
Yes, it's non-fatal for both units and non-units. I thought it made sense to do so instead of quitting early.

Reply:
Ah right, both cases, derp. I think it's fine to do, just wanted to ensure I understood.